tests/deprecated_api/test_remove_1-6.py

```diff
@@ -146,23 +146,23 @@ def training_step(self, *args):
 def test_v1_6_0_datamodule_lifecycle_properties(tmpdir):
     dm = BoringDataModule()
     with pytest.deprecated_call(match=r"DataModule property `has_prepared_data` was deprecated in v1.4"):
-        dm.has_prepared_data
+        pass
```

Comment:
we probably shale have some asserts?

Reply:
there is an assert here with the context manager! what's the point of this PR?

Reply:
yes but still we can make assert that the call is valid
```suggestion
        assert isinstance(dm.has_prepared_data, bool)
```

Reply:
the purpose of the test is to check that the deprecation warning is raised, not that the property is a boolean

Reply:
ok, then simple
```suggestion
        _ = dm.has_prepared_data
```